### PR TITLE
Bump pgautofailover all version to 1.3.1

### PR DESCRIPTION
**Pull request by JelteF**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,16 @@
+pg-auto-failover (1.3.citus-1) stable; urgency=low
+
+  * Fix systemd integration [#212]
```
Comment (hanefi) [inline debian/changelog]: We have empty lines between changelog items.

File: pg-auto-failover.spec [hunk 2]
```diff
@@ -9,11 +9,11 @@ Summary:	Postgres extension for automated failover and high-availability
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	%{sname}_%{pgmajorversion}
 Conflicts:	%{sname}_%{pgmajorversion}
-Version:	1.2
+Version:	1.3.citus
```
Comment (hanefi) [inline pg-auto-failover.spec]: Is this actually intended? Do we want this for consistency between packages for different extensions?

**Review by hanefi — CHANGES_REQUESTED** [review]

File: pg-auto-failover.spec (hunk 2 above)
Comment (JelteF) [reply pg-auto-failover.spec]: Yes I guess the reason is that it's clear where the package comes from, i.e. built by us not pgdg.

File: debian/changelog (hunk 1 above)
Comment (JelteF) [reply debian/changelog]: This is automatically filled by `update_os_package.pl`. This is simply the way the changelog is formatted on the pg_auto_failover repo.

Comment (DimCitus) [reply debian/changelog]: The debian changeling is supposed to contain entries relevant to the debian packaging, not to the software being packaged. So you could have entries about the improvements you made in the debian packaging, or just one line mentioning it's an official release of the software...

File: pg-auto-failover.spec [hunk 3]
```diff
@@ -9,11 +9,11 @@ Summary:	Postgres extension for automated failover and high-availability
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	%{sname}_%{pgmajorversion}
 Conflicts:	%{sname}_%{pgmajorversion}
-Version:	1.2
+Version:	1.3.1.citus
```
Comment (DimCitus) [inline pg-auto-failover.spec]: Is there a reason to suffix with Citus here?

File: debian/changelog (hunk 1 above)
Comment (DimCitus) [reply debian/changelog]: See https://www.debian.org/doc/debian-policy/ch-source.html#debian-changelog-debian-changelog for more information and details.

File: pg-auto-failover.spec (hunk 2 above)
Comment (DimCitus) [reply pg-auto-failover.spec]: Yes but it's confusing, is it the package for pg_auto_failover with Citus support?

File: pg-auto-failover.spec (hunk 3 above)
Comment (JelteF) [reply pg-auto-failover.spec]: Yes, the reason is to easily differentiate our packages from ones installed through other means, i.e. pgdg or OS packages.

File: pg-auto-failover.spec (hunk 2 above)
Comment (JelteF) [reply pg-auto-failover.spec]: That's true, I removed it now

File: debian/changelog (hunk 1 above)
Comment (hanefi) [reply debian/changelog]: I suggest we have the usual single entry here:
```
  * Official release of 1.3.1
```

We will be missing an entry for 1.3 but I guess this is the way it is here, and we are missing several other changelog sections as well.

File: pg-auto-failover.spec [hunk 4]
```diff
@@ -82,6 +82,9 @@ PATH=%{pginstdir}/bin:$PATH
 
 
 %changelog
+* Thu May 7 2020 - Jelte Fennema <Jelte.Fennema@microsoft.com> 1.3.1-1
+- Official 1.3.1 release of pg_auto_failover
```
Comment (hanefi) [inline pg-auto-failover.spec:86]: I would have wanted to see a consistent item in here such as:
```
  - Official release for 1.3.1
```

File: debian/changelog [hunk 5]
```diff
@@ -1,3 +1,9 @@
+pg-auto-failover (1.3.1-1) stable; urgency=low
+
+  * Official 1.3.1 release of pg_auto_failover
```
Comment (hanefi) [inline debian/changelog:3]: I would have wanted to see a consistent item in here such as:
```
  - Official release for 1.3.1
```

**Review by hanefi — APPROVED** [review]
After 2 simple changes in changelog items for consistency between all sections, we can merge this

File: pg-auto-failover.spec (hunk 4 above)
Comment (JelteF) [reply pg-auto-failover.spec:86]: I think it's better with the name of the tool in there too. 
I've modified `update_os_package.pl` to do this automatically for new versions:
https://github.com/citusdata/tools/pull/69/files#diff-99ba4ed771cad68c7bae27b53c0c4aa3R91

**Pull request merged**